gdi/opentelemetry/install-k8s-addon-eks.rst

```diff
@@ -1,34 +1,32 @@
 .. _install-k8s-addon-eks:
 
 *******************************************************************************
-Splunk Distribution of the OTel Collector Add-on for AWS EKS
+Splunk Distribution of the OpenTelemetry Collector as EKS Add-on 
 *******************************************************************************
 
 .. meta::
-    :description: AWS Add-on for EKS
+    :description: Kubernetes (EKS Add-on)
```

Comment:
Naming things is hard. I'll offer my input and some of the future products we will (likely) end up supporting. Let's collaborate as needed. 
```suggestion
    :description: Kubernetes via the EKS Add-on installation
```

We will have very similar to support in the future to support which might be named like this.
- Kubernetes via the GCP Add-on installation
- Kubernetes via the Azure Add-on installation